compiler/rustc_monomorphize/src/collector.rs

```diff
@@ -857,7 +857,8 @@ impl<'a, 'tcx> MirVisitor<'tcx> for MirUsedCollector<'a, 'tcx> {
         };
 
         match terminator.kind {
-            mir::TerminatorKind::Call { ref func, ref args, ref fn_span, .. } => {
+            mir::TerminatorKind::Call { ref func, ref args, ref fn_span, .. }
+            | mir::TerminatorKind::TailCall { ref func, ref args, ref fn_span } => {
```

Comment:
Once https://github.com/rust-lang/rust/pull/113128 lands, the mentioned_items pass will need a similar patch.

Reply:
I'm assuming you meant #122568?

Reply:
Eh. Yes. Sorry.^^